src/components/mdIconCSS/mdIconCSS.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <i class="md-icon-css" :class="[themeClass, cls]">
+     <slot></slot>
+  </i>
+</template>
+
+<style lang="scss" src="./mdIconCSS.scss"></style>
+
+<script>
+  import theme from '../../core/components/mdTheme/mixin';
+
+  export default {
+    props: {
+      cls: String
```

Comment:
You actually don't need to use `cls` as a property. According to [the docs](https://vuejs.org/v2/guide/class-and-style.html#With-Components) classes put on a component get combined and rendered correctly. So, this:

```html
<md-icon-css class="md-primary fa fa-fw fa-home"></md-icon-css>
```

will get rendered as:

```html
<i class="md-icon-css md-primary fa fa-fw fa-home"></i>
```